### DISCUSSION TITLE
Using discussions instead of issues for Q&A and Ideas

### DISCUSSION THREAD

**Discussion by zain-sohail** [momentoscope/hextof-processor#100] [Ideas]
We can move the issues which are not really issues to discussion. 

User questions for example can also stay up all the time on discussions instead of them dissapearing after their issue is fixed.

Wha do you think?

**Comment by steinnymir** [comment]
this is a really good idea! 
like this we can keep track of issues and solve them when needed, and not overwhelm them with ideas and discussions!
